src/sth_helper.js

```diff
@@ -117,9 +117,9 @@
       return sthConfig.UNICA_CORRELATOR.NOT_AVAILABLE;
     } else {
       return getHashCode('from: ' + request.info.remoteAddress +
-                         ':' + request.info.remotePort +
-                         ', method: ' + request.method.toUpperCase() +
-                         ', url: ' + request.url.path);
+      ':' + request.info.remotePort +
```

Comment:
Not currently using any linting processing (we have an issue for that https://github.com/telefonicaid/IoT-STH/issues/8) I'll fix it as soon as the pending code lands to assure that code styles are correct :) Are you fine with this approach?